README.md

```diff
@@ -91,15 +91,22 @@ Extensions are the core way that you can extend and customize osquery. At Facebo
 The same Thrift bindings can be used to create a Python client for the `osqueryd` or `osqueryi`'s extension socket. There are helper classes provided that spawn an ephemeral osquery process for consecutive or long running client instances.
 
 ```python
+#!/usr/bin/env python3
+
 import osquery
+import pprint
+import osquery.management as management
+
+# Override default location: /usr/bin/osqueryd
+management.LINUX_BINARY_PATH = "/usr/local/bin/osqueryd"
```

Comment:
I think `/usr/bin/osqueryd` is the default if you install osquery through a package and `/usr/local/bin/osqueryd` is the default if you use `make install` (or through the hosted tgz bundle).

My recommendation is to make the change in the example, but rather keep this line commented-out for people who install via `make install`.

Reply:
It's also possible to define the path, as a parameter to the `SpawnInstance()` method - thus I will close this PR.